port_ocean/ocean.py

```diff
@@ -29,6 +30,13 @@ def __init__(
     ):
         initialize_port_ocean_context(self)
         self.fast_api_app = app or FastAPI()
+        self.fast_api_app.add_middleware(
```

Comment:
Do we really need it?
U added a middleware for your case

Reply:
right, no need